src/hooks/useDrag.tsx

```diff
@@ -197,6 +235,17 @@ const isUpdateHomepageType = (
       !Number.isNaN(Number(possibleCardIndex))
     )
   }
+
+  if (typeof value === "string" && value.startsWith("infocolInfobox-")) {
+    const valArr = value.split("-")
+    const possibleInfoboxIndex = valArr[1]
+    return (
+      valArr.length === 2 &&
```

Comment:
Hmm why suddenly go 2 here ah? could we constant it vs using magic numbers?

Reply:
Hmm i don't feel this is a magic number. If read within the context, we are doing a split on line 240. Hence, the expectation is to receive an array of length 2